sdk/include/opentelemetry/sdk/trace/processor.h

```diff
@@ -47,7 +47,7 @@ class SpanProcessor
    * timeout is applied.
    */
   virtual bool ForceFlush(
-      std::chrono::microseconds timeout = std::chrono::microseconds::max()) noexcept = 0;
+      std::chrono::microseconds timeout = (std::chrono::microseconds::max)()) noexcept = 0;
```

Comment:
What is this change for?

Reply:
This is to resolve the `NOMINMAX` compilation issue. Visual Studio and Windows SDK is notoriously bad for defining a macro for `max` like `#define max` - which breaks any C++ Standard Library methods that use `max` method 😄

Reply:
https://stackoverflow.com/questions/13416418/define-nominmax-using-stdmin-max

Cit. `If you're really desperate, put parentheses around the function names...`, - we were desperate 😁